@client/edit_comment.coffee

```diff
@@ -122,51 +126,78 @@ window.EditComment = ReactiveComponent
             marginTop: 10
             marginBottom: 10
             marginLeft: 60 
-          for error in @local.errors
-            DIV null, 
+          for error, error_index in @local.errors
+            DIV
+              key: 'error' + error_index
               I
                 className: 'fa fa-exclamation-circle'
                 style: {paddingRight: 9}
 
               SPAN null, error
 
       if permitted > 0
-
```

Comment:
Almost all the change here is a trivial indentation, down to "DIV key: 'sign_checkbox'"